torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1872,10 +1875,10 @@ TensorExprKernel::TensorExprKernel(
       symbolic_shape_inputs_(std::move(symbolic_shape_inputs)),
       custom_lowerings_(std::move(custom_lowerings)),
       pre_alloc_(pre_alloc),
-      kernel_func_name_(kernel_func_name),
+      kernel_func_name_(std::move(kernel_func_name)),
       symbolic_strides_(std::move(symbolic_strides)) {
   optimizeOwningGraph();
-
+  // NOLINTNEXTLINE
```

Comment:
What's the lint failure on the next line?

Reply:
Fixed